charts/mapgl-js-api/values.yaml

```
@@ -64,6 +67,9 @@ env:
   MAPGL_IMMERSIVE_TILESET: web_immersive
   MAPGL_TRAFFICSERVER: https://traffic-proxy.ingress.host
   MAPGL_FLOORSSERVER: https://floors-api.ingress.host
+  MAPGL_STYLESERVER: https://styles.api.2gis.com
+  MAPGL_ICONSPATH: https://disk.2gis.com/styles/assets/icons
+  MAPGL_MODELSPATH: https://disk.2gis.com/styles/assets/models
```

```suggestion
  MAPGL_MODELSPATH: https://styles.ingress.host/styles/assets/models
```

До on-cloud'овых сервисов в on-premise обычно доступа нет, незачем их упоминать